pyscf/neo/ase.py

```diff
@@ -37,6 +38,17 @@ def stable_opt_internal(mf):
     return mf
 
 class Pyscf_NEO(Calculator):
+    """
+    init_guess:
+
+    None: The default option. use the default initial guess
+    'last_step': use both electronic and nuclear density 
+                 matrices from the last step as the initial guess
+    'elec_only': use only electronic density matrix 
```

Comment:
Consider changing the name. It is a variant of `last_step` but with electronic dm only.